server/routes/schedules.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+const {
+  getAllSchedules,
+  getScheduleById,
+  addSchedule,
+  updateSchedule,
+  deleteSchedule,
+} = require("../controller/schedulesController");
+
+router.use(express.json());
+
+router.get("/", getAllSchedules);
```

Comment:
all과 명사+복수형은 비슷한 의미를 가지고 있다고 볼 수 있어서요!
아래와 같이 표현해줘도 저는 충분할 것 같아요~
```suggestion
router.get("/", getSchedules);
```